environments/kc-2023-chi/terragrunt.hcl

```diff
@@ -4,23 +4,23 @@ include "root" {
 }
 
 terraform {
-  source = "somegit"
+  source = "https://github.com/thedevelopnik/kuberoke-registration.git//terraform"
```

Comment:
Let's have this point to a ref of 1.0.0 and release the tag once we merge this PR and your other PR.

Reply:
now it should point to the new kuberoke org repo

Reply:
Done!